CHANGELOG.md

```diff
@@ -16,11 +16,18 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 - .NET Framework only, `Grpc.Core.Api` updated from `2.57.0` to `2.58.0`.
 - .NET only, `OpenTelemetry.Instrumentation.EntityFrameworkCore` updated
   from `1.0.0-beta.7` to `1.0.0-beta.8`.
+- .NET only, `OpenTelemetry.Instrumentation.AspNetCore` updated
+  from `1.5.1-beta.1` to `1.6.0-beta.2`.
+- `OpenTelemetry.Instrumentation.GrpcNetClient`,
+  `OpenTelemetry.Instrumentation.Http`,
+  and `OpenTelemetry.Instrumentation.SqlClient` updated from `1.5.1-beta.1` to `1.6.0-beta.2`.
 
 ### Deprecated
 
 ### Removed
 
+- Plugins, removed supported for `OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreMetricsInstrumentationOptions`.
```

Comment:
Ideally we provide some information about what needs to be changed for anyone implementing a plugin leveraging that.

Reply:
The solution should be obvious for the plugins authors when they update dependency to 1.6.0-beta.2.
It will not compile as the options class is now internal.

Reply:
I am changing wording to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/a6e4c34a3dc6f05c4cbdfc13324ea5e730307037/CHANGELOG.md#removed-1

Nobody was complaining for now. Plugins are advanced cases, the requierments is to always build it based on the refereneced versions. If plugin for older versions is working with the newer, it is just by accident, and at least bad practice for plugins creator.

```suggestion
- Removed support for `OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreMetricsInstrumentationOptions`
  for plugins.
```